src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.model;
+
+import java.util.Objects;
+
+public class Fruit {
+    private Integer amount;
```

Comment:
let's leave for now, but you can store information about objects quantity inside one of these objects